accelerator/cli_info.h

```diff
@@ -73,9 +75,25 @@ static struct ta_cli_argument_s {
                           {"verbose", VERBOSE, "Enable logger", NO_ARG}};
 
 static const int cli_cmd_num = sizeof(ta_cli_arguments_g) / sizeof(struct ta_cli_argument_s);
-void ta_usage();
+
+static inline void ta_usage() {
```

Comment:
Why?

Reply:
I thought it is redundant to use an extra source file to implement only one function, so I merge them into a single header file.